.github/workflows/sync_images.yml

```
@@ -1,50 +1,49 @@
+jobs:
+  sync_images:
+    name: Sync Images
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)